gm4_end_fishing/data/gm4_end_fishing/functions/main.mcfunction

```diff
@@ -6,6 +6,6 @@ execute as @a[tag=gm4_has_enderpuff] run function gm4_end_fishing:enderpuff/proc
 scoreboard players reset @a gm4_ef_sneak
 
 #fix enderpuff head
-execute as @e[type=item,nbt={Item:{id:"minecraft:player_head",Count:1b,tag:{SkullOwner:{Id:[I;2008953110,-1021689564,-2086641039,1315516205]}}}}] run data merge entity @s {Item:{tag:{CustomModelData:14,gm4_end_fishing:{enderpuff:1b},display:{Name:'{"translate":"%1$s%3427655$s","with":["Enderpuff",{"translate":"item.gm4.enderpuff"}],"italic":false}'}}}}
+execute as @e[type=item,nbt={Item:{id:"minecraft:player_head",Count:1b,tag:{SkullOwner:{Id:[I;-749248143,1561284060,-1964497486,1336566238]}}}}] run data merge entity @s {Item:{tag:{CustomModelData:14,gm4_end_fishing:{enderpuff:1b},display:{Name:'{"translate":"%1$s%3427655$s","with":["Enderpuff",{"translate":"item.gm4.enderpuff"}],"italic":false}'}}}}
```

Comment:
This should be moved into a function as we need to keep the old repair command for a couple of versions. I'd be best to pull the `Id` into storage and then run the appropriate command based on that.